Example/Pods/Pods.xcodeproj/project.pbxproj

```diff
@@ -1008,7 +1005,7 @@
 				GCC_PREFIX_HEADER = "Target Support Files/Nimble/Nimble-prefix.pch";
 				INFOPLIST_FILE = "Target Support Files/Nimble/Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```

Comment:
Lets keep this as low as possible to achieve the revolving gallery goal. 9.0? Even less if possible.

Reply:
We'll also need to update the podspec to reflect this change as well.

Reply:
9.0 will be the lowest, just updated it